### PR TITLE
Count the coverage in E2E tests and Plugin tests

### PR DESCRIPTION
**Pull request by kezhenxu94**

### CODE REVIEW

**Comment by wu-sheng** [comment]
A reminder, what happens if plugin tests don't run? I assume it will affect the coverage rate.

**Comment by wu-sheng** [comment]
Do we need to run them always to keep the coverage reasonable? Once this is done, I think you should close #2438 due to this should provide a very high coverage rate.

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/ProfileTaskCreationResult.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package org.apache.skywalking.oap.server.core.profile.entity;
+package org.apache.skywalking.oap.server.core.query.entity;
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/ProfileTaskCreationResult.java:19]: I think this is mischange?

Comment (kezhenxu94) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/ProfileTaskCreationResult.java:19]: > I think this is mischange?

The GraphQL resolver will generate some classes for the query entities, which breaks the Jacoco agent, we have to exclude these classes in Jacoco, though I can have another rules to exclude this particular class (`ProfileTaskCreationResult`), but I think it belongs package `org.apache.skywalking.oap.server.core.query.entity`, as other similar query entities

Comment (wu-sheng) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/entity/ProfileTaskCreationResult.java:19]: Good to me.

**Pull request closed without merging**